src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/DriveRightwardOnly.java

```diff
@@ -0,0 +1,34 @@
+package edu.greenblitz.tobyDetermined.commands.swerve;
```

Comment:
I think the name of the function is not right - It has to mention something about the ultrasonic sensor as falling-off defence.
AND DOC EVERYTHING!